src/main/java/com/jmb/springfactory/model/dto/PermisionDto.java

```diff
@@ -8,5 +8,6 @@
 public class PermisionDto extends BaseDto {
 
   private static final long serialVersionUID = -3597965386587437015L;
+  private Integer id;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "id" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)